src/astro.js

```diff
@@ -4,28 +4,44 @@ import {
   telephotoCamera,
   calibrateGotoCmd,
   startGotoCmd,
+  stopGotoCmd,
   binning2x2,
   fileTiff,
   takeAstroPhotoCmd,
   takeAstroDarkFramesCmd,
   darkGainDefault,
   utcURL,
+  timeZoneURL,
   stopAstroPhotoCmd,
   rawPreviewContinousSuperimpose,
   queryShotFieldCmd,
+  astroAutofocusCmd,
   setRAWPreviewCmd,
 } from "./api_codes.js";
-import { nowUTC, nowLocal, nowLocalFileName } from "./api_utils.js";
+
+import { nowUTC, nowLocalFileName } from "./api_utils.js";
 
 /**
- * 4.1.1 UTC+0 time
+ * 4.1.0 UTC+0 time
```

Comment:
For this library, the numbers for the function should be the same numbers used in the Dwarf API docs, so people can quickly cross reference code in this library with the code in Dwarf API.  

Which version of the Dwarf API Docs are you using? I used v1.0, and setting UTC is 4.1.1. There is no setting timezone.

Reply:
###

Reply:
Yes it's the same, but this function is not documented, (see the request on the fly) so add a number not used maybe need to add : not documented

Reply:
Add `not documented` is a good idea.